Tasks/PerformActionOnMachineGroup/AzureHelper.ps1

```diff
@@ -60,6 +60,32 @@ function Restart-MachineInProvider
 
 function Initialize-AzureHelper
 {
+    Write-Verbose "Entering in azure-initializer" -Verbose
```

Comment:
Import-AzurePowerShellModule this is a commandlet in the internal dll. Use this just to be sure that the module is loaded